open_spiel/games/german_whist_foregame/german_whist_foregame.h

```diff
@@ -0,0 +1,130 @@
+#ifndef OPEN_SPIEL_GAMES_GERMAN_WHIST_FOREGAME_H
+#define OPEN_SPIEL_GAMES_GERMAN_WHIST_FOREGAME_H
+
+#include <array>
+#include <memory>
+#include <string>
+#include <vector>
+#include <random>
+#include <x86intrin.h>
```

Comment:
Is this a standard header?

Is it used anywhere in this file? Code submitted to the master repos should be portable as we also support MacOS run on different architectures.

Reply:
If you are using x86 architecture optimizations, what I suggest is that you keep this file as it is now (i.e. for your research) and submit a different one that is completely general and can run on non-x86 architectures.